### DISCUSSION TITLE
Up next playlist

### DISCUSSION THREAD

**Discussion by allasca** [HemantKArya/BloomeeTunes#40] [General]
when playing audio from YT, up next playlist contain all result from search. how to like more YT, something like next different song?

```dart
var video = yt.videos.get('https://youtube.com/watch?v=Dpp1sIL1m5Q');
var relatedVideos = await yt.videos.getRelatedVideos(video); // video must be a Video instance.
print(relatedVideos); //prints the list of related videos

// to get the next page of related videos
relatedVideos = await relatedVideos.nextPage();
```

**Comment by HemantKArya** [comment]
Hi @allasca thanx for noticing this 😊. Yeah, It will be great, and I think we should develop more robust API for that so that if the source is different than it loads music from the same source. I was thinking about this feature sometime before the only problem was that when playing from youtube it could be headache if it returns some random youtube videos which are not songs/musical audios.
So, yeah, we have to make and API code which return songs related to current songs no matter what the source is.

One of the things I guess is to use API like Last.FM and other scrobblers that provides related songs feature. or maybe something else. We need more discussion ideas related to this problem to make a fully working API.
thanx for opening it btw.
